src/main/java/ru/yandex/qatools/allure/jenkins/AllureReportPublisher.java

```diff
@@ -163,7 +166,7 @@ private void generateReport(@Nonnull List<FilePath> resultsPaths, @Nonnull Run<?
         configureJdk(launcher, listener, buildEnvVars);
         final AllureCommandlineInstallation commandline = getCommandline(launcher, listener, buildEnvVars);
 
-        final FilePath reportPath = workspace.child("allure-report");
+        final FilePath reportPath = workspace.createTempDir(ALLURE_PREFIX, ALLURE_REPORT_SUFFIX);
```

Comment:
but there's a request to specify report folder and not delete it https://github.com/jenkinsci/allure-plugin/issues/114